### PR TITLE
Remove MorganState and SetMorganMask (non-existent functions)

### PR DESCRIPTION
**Pull request by nashley**
Fixes Morgan

### CODE REVIEW

File: world/map/npc/items/check_wand.txt [hunk 1]
```diff
@@ -48,8 +48,7 @@ L_Loop:
 L_SetWand:
     set @Wand, $@WandsPwr[@wand_loop];
     set @WandID, $@WandsAnim[@wand_loop];
-    callfunc "MorganState";
-    if (@morgan == 2)
+    if (@QL_MORGAN == 2)
```
Comment (mekolat) [inline world/map/npc/items/check_wand.txt]: not `@QL_MORGAN`. please remove the @

**Comment by mekolat** [comment]
@wushin this needs to go in BEFORE the release

**Comment by mekolat** [comment]
@nashley squash `Merge pull request #1 from mekolat/npc/morgan_fix` into `fix liana` and `Removed stray set` into `Morgan's dialog is fixed`

**Comment by nashley** [comment]
@mekolat I will once I rewrite her dialog.
Unless you want me to now.

**Comment by mekolat** [comment]
@nashley make another PR if you want to fix the dialog so that wushin can merge the bugfix even if you are not done with the dialog

**Comment by nashley** [comment]
@wushin @mekolat Done.

**Pull request merged**